.github/workflows/release.yml

```
@@ -112,33 +112,54 @@ jobs:
 
   micro-services:
     name: Build micro services
+    runs-on: ubuntu-22.04
     needs: [verify-inputs, prepare-release-branch]
-    uses: ./.github/workflows/build-micro-service-manual.yml
-    secrets: inherit
     strategy:
       matrix:
-        service:
-          [join-service, key-service, verification-service, qemu-metadata-api]
-    with:
-      microService: ${{ matrix.service }}
-      imageTag: ${{ inputs.version }}
-      version: ${{ needs.verify-inputs.outputs.WITHOUT_V }}
-      ref: ${{ needs.verify-inputs.outputs.RELEASE_BRANCH }}
-      release: true
+        koTarget:
+          [./joinservice/cmd, ./keyservice/cmd, ./verify/cmd, ./operators/constellation-node-operator]
+        include:
+          - koTarget: ./joinservice/cmd
+            name: join-service
+          - koTarget: ./keyservice/cmd
+            name: key-service
+          - koTarget: ./verify/cmd
+            name: verification-service
+          - koTarget: ./operators/constellation-node-operator
+            name: node-operator
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          ref: ${{ github.head_ref }}
```

```suggestion
          ref: ${{ needs.verify-inputs.outputs.RELEASE_BRANCH }}
```

I think this is what we actually want here